lib/getOriginalConstructor/README.md

```diff
@@ -0,0 +1,13 @@
+# getOriginalConstructor
+
+This file exposes a utility function to get a reset version of a primitive
+constructor, in the event that a client has overridden or polyfilled prototype
```

Comment:
You may want to reword this. As an OS repo, "site" is probably more appropriate than "client".